.azure-pipelines.yml

```diff
@@ -152,10 +152,11 @@ jobs:
     parameters:
       packages: virtualenv
   - template: tools/ci/azure/install_certs.yml
+  - template: tools/ci/azure/install_edge.yml
   - template: tools/ci/azure/update_hosts.yml
   - template: tools/ci/azure/update_manifest.yml
-  - script: python ./wpt run --yes --no-manifest-update --install-fonts --manifest MANIFEST.json --metadata infrastructure/metadata/ --webdriver-arg=--verbose --log-tbpl $(Build.ArtifactStagingDirectory)/edge.tbpl.log --log-tbpl-level info edge_webdriver infrastructure/
-    displayName: 'Run tests (Edge)'
+  - script: python ./wpt run --yes --no-manifest-update --install-fonts --manifest MANIFEST.json --metadata infrastructure/metadata/ --webdriver-arg=--verbose --log-tbpl $(Build.ArtifactStagingDirectory)/edge.tbpl.log --log-tbpl-level info --channel canary edgechromium infrastructure/
```

Comment:
Does the new WebDriver binary also need the `--verbose` arg? If not needed or if not even recognized, it can be dropped here and elsewhere.

Reply:
It does support --verbose flag, but it's not need at the moment as the runs are pretty stable, so i've removed it.